highton/call_mixins/update_call_mixin.py

```diff
@@ -14,9 +15,11 @@ def update(self):
         :return:
         :rtype: response
         """
+        this = deepcopy(self)
+        this.party = None
```

Comment:
This would update ALL models which have the update_call_mixin... 
not all models support parties anyway.
Maybe there is a better approach?

Reply:
True... buuu~t, since it's a deep copy, it wouldn't actually change the model in Highrise. As long as this changes is guaranteed never to be persisted, I thought it might be fine. 

It's super not elegant, I'll give you that.

Reply:
I mean that if you want to update any model it would send party=None field... e.g. in custom-fields.
You would send an unexpected field in models which dont have the party field

Reply:
I mean we could make it even more ugly by checking via `hasattr` first..?

Reply:
The problem is... that this doing things in which are made letter in a different context.
This function is too broad to have such a concrete condition.
I would appreciate it, if we use the last scope, which can be changed e.g. the Deal Model which should override the update method or something

Reply:
Sure, we can do that instead.